docs/demo.mdx

```diff
@@ -0,0 +1,370 @@
+---
+title: Internal Demo Page for Styling
+description: This page is for internal use only.
+---
+
+<head>
+  <meta name="robots" content="noindex, nofollow" />
+  <title>Detox Interal Demo Page</title>
+</head>
+
+import HomepageHeader from '@site/src/components/Homepage/Header/HomepageHeader';
+
+import '@site/src/css/custom.css';
+import '@site/node_modules/infima/dist/css/default/default.css';
+
+import FlavorizedCodeBlock from '@site/src/components/FlavorizedCodeBlock';
+
+import CodeBlock from '@theme/CodeBlock';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+:::note
+
+This page is for internal use only. If you are a contributor to the Detox project, you can use this page to test your changes to the website documentation pages
+styling. If you are not a contributor, you can ignore this page. Follow this link to the [Detox contributing Page](https://wix.github.io/Detox/docs/contributing)
```

Comment:
I'd suggest to return to the home page if the visitor is not a contributor.

Reply:
Good point. Thanks. Edited to return to Detox Home Page.